src/_css/NavBarComponent.scss

```diff
@@ -132,3 +136,89 @@
   margin-top: 1.5rem;
   font-size: 15px;
 }
+
+#analytics-state-selection {
+  .ui.grid {
+    .column.row {
+      .column {
+        margin-bottom: 15px;
+      }
+    }
+  }
+}
+
+#asset-links-section {
```

Comment:
Why are you using `id` instead of `class` for styling?  `id` will make it impossible to reuse the styles unless you copy-paste them somewhere else